server/graphql/v2/object/Host.js

```diff
@@ -393,53 +466,44 @@ export const GraphQLHost = new GraphQLObjectType({
           }
 
           const collectiveIds = isEmpty(args.collectiveAccountIds)
-            ? undefined
+            ? [null]
```

Comment:
No, this is because the sequelize query replacement with arrays requires at least one value, even if null, to generate a query with correct syntax. This null value will shortcircuit the query and ignore the collective filter.